gulpfile.js

```diff
@@ -245,6 +245,18 @@ gulp.task('test', ['browserify-tests', 'js'], () => {
     .pipe(reload());
 });
 
+gulp.task('test:unit', ['browserify-tests'], () => {
```

Comment:
Duplicate of `test`? Maybe `test` should run `test:unit` and `test:selenium`?

Reply:
I think `test` is still being used to run locally, in that case it would still need to run `js` to ensure code is up-to-date. Could renamed it to `local:test` or `local:test:unit`.

Reply:
`test` is run locally during development. Happy for it to be renamed `local:test:unit` for clarity.